.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "overrides": [
+    {
+      "files": "*.sol",
+      "options": {
+        "printWidth": 500,
```

Comment:
I think there need to be at least some reasonable upper limit. 120 is too low, but 500 is absurd haha.